activiti-cloud-notifications-graphql-service/starter/src/test/resources/application.properties

```diff
@@ -17,4 +17,6 @@ keycloak.auth-server-url=http://localhost:8180/auth
 keycloak.realm=activiti
 keycloak.resource=activiti
 keycloak.public-client=true
-keycloak.cors=true
\ No newline at end of file
+keycloak.cors=true
+
+activiti.cloud.notification.uri-prefix=
```

Comment:
You should not need to have to define this in test properties if there is default properties source defined for each service